docs/pages/versions/unversioned/sdk/facebook.md

```diff
@@ -28,14 +28,14 @@ Then follow these steps based on the platforms you're targetting. This will need
 
 - **The Expo client app**
 
-  - Add `host.exp.Exponent` as an iOS _Bundle ID_. Add `rRW++LUjmZZ+58EbN5DVhGAnkX4=` as an Android _key hash_. Your app's settings should end up including the following under "Settings > Basic":
+  - In the Expo Client, all of your Facebook API calls will be made with Expo's Facebook App ID. To use your app's own Facebook App ID (and thus send related app events to your Facebook developer console), you'll need to [build a standalone app](../../distribution/building-standalone-apps/).
```

Comment:
Do you think the ramifications you mentioned in

> > What are the consequences of using Expo's Facebook app for all API calls in Expo Client?
> 
> The main drawback is that they won't see app events logged, since their app isn't being initialized. Possibly also ramifications for facebook ads, since that relies on facebook app id as well

are worth listing here?

Reply:
added here and to facebook-ads documentation